x/programs/runtime/config.go

```diff
@@ -0,0 +1,187 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package runtime
+
+import "github.com/bytecodealliance/wasmtime-go/v13"
+
+const (
+	defaultMaxWasmStack    = 256 * 1024 * 1024 // 256 MiB
+	defaultWasmThreads     = false
+	defaultFuelMetering    = true
+	defaultWasmMultiMemory = false
+	defaultWasmMemory64    = false
+	defaultLimitMaxMemory  = 16 * 64 * 1024 // 16 pages
+)
+
+// TODO: review Cranelift fine tune knobs when exposed
```

Comment:
Can we create an issue to do this (I think its been merged and released)?

Reply:
I did update to the latest and added a default for `enable_nan_canonicalization`. But will add an issue for digging further.